cmd/mapcidr/main_test.go

```
@@ -324,3 +326,66 @@ func TestProcess(t *testing.T) {
 
 	}
 }
+
+func TestOutputItemsSuppressesStdoutWhenFileSet(t *testing.T) {
+	// Regression for #826: -o should not also emit results on stdout.
+	path := t.TempDir() + "/out.txt"
+	f, err := os.Create(path)
+	require.NoError(t, err)
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)